__exekutir__/roles/kommandir_workspace_setup/tasks/main.yml

```diff
@@ -123,8 +123,9 @@
   git:
     repo: "{{ item.repo }}"
     dest: "{{ kommandir_workspace }}/cache/{{ item.cachepath }}"
-    # Shallow clone is much faster, two commits will cover a merge-commit
-    depth: "{{ item.depth | default(2) }}"
+    # Shallow clone is much faster, two commits will cover a merge-commit and will be set per repo.
+    # default(omit) to workaround failure with python-bugzilla repo
+    depth: "{{ item.depth | default(omit) }}"
```

Comment:
@edsantiago docs ref: http://jinja.pocoo.org/docs/2.9/templates/#list-of-control-structures
(I keep a copy under my pillow)